docs/docs/widgets/textarea.md

```diff
@@ -2,45 +2,47 @@
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Component Specific Actions (CSA)
 
 Following actions of the **Textarea** component can be controlled using Component-Specific Actions(CSA):
 
-| <div style={{ width:"100px"}}> Actions  </div>   | <div style={{ width:"135px"}}> Description </div> | <div style={{ width:"135px"}}> How To Access </div> |
-| :----------- | :----------- |:---------|
-| setText | Sets the text on the text area component via a component-specific action within any event handler.|  Employ a RunJS query to execute component-specific actions such as `await components.textarea1.setText('this is a textarea')`. |
-| clear | Clears the value from the text area component via a component-specific action within any event handler.| Employ a RunJS query to execute component-specific actions such as `await components.textarea1.clear()`. |
+| <div style={{ width:"100px"}}> Actions </div> | <div style={{ width:"135px"}}> Description </div>                                                       | <div style={{ width:"135px"}}> How To Access </div>                                                                            |
+| :-------------------------------------------- | :------------------------------------------------------------------------------------------------------ | :----------------------------------------------------------------------------------------------------------------------------- |
+| setText                                       | Sets the text on the text area component via a component-specific action within any event handler.      | Employ a RunJS query to execute component-specific actions such as `await components.textarea1.setText('this is a textarea')`. |
+| clear                                         | Clears the value from the text area component via a component-specific action within any event handler. | Employ a RunJS query to execute component-specific actions such as `await components.textarea1.clear()`.                       |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Exposed Variables
 
-| Variables | Description | How To Access |
-|:---------:|:-----------:|:-------------:|
-| <div style={{ width:"100px"}}> value </div> | This variable holds the value entered in the text area component. | Access the value dynamically using JS. For example, `{{components.textarea1.value}}` |
+| Variable                                                          | Description                                                                          | How To Access |
+| :---------------------------------------------------------------- | :----------------------------------------------------------------------------------- | :------------ |
+| <div style={{ width:"100px"}}> value </div>                       |
```

Comment:
Hey, no changes were expected in this table but still, some changes are made and the table is breaking because of that, can you please correct it?

![image](https://github.com/user-attachments/assets/df0dfc09-7f88-4086-b004-1aa161867daa)